src/components/tables/DataTable.js

```diff
@@ -141,10 +160,21 @@ export default {
     },
     search () {
       this.page = 1
+    },
+    page () {
+      this.update()
```

Comment:
Are you saying it gets called multiple times in the code, or when using it? As far as I can tell it only triggers once when paging/changing rows per page. The update call in the rowsPerPage watch is so that it emits update if it's already on page 1, since the page watcher won't active then.